appveyor.yml

```diff
@@ -34,17 +38,29 @@ branches:
 cache:
   - C:\Users\appveyor\.cache -> appveyor-clean-cache.txt
   - C:\Users\appveyor\AppData\Local\pip\Cache -> appveyor-clean-cache.txt
+  - '${HOME}/cache -> appveyor-clean-cache.txt'
 
 init:
   - ps: $Env:path = "C:/Enthought/edm;" + $Env:path
   - ps: md C:/Users/appveyor/.cache -Force
+  # note that on osx the .cache folder is owned by root
+  # and we cannot create files there.
+  - sh: export PATH="${PATH}:/usr/local/bin"
+  - sh: export XDG_CACHE_HOME="${HOME}/cache"
+  - sh: export LC_ALL=en_US.UTF-8
+  - sh: export LANG=en_US.UTF-8
+  - sh: mkdir -p "${HOME}/cache/download"
+  - sh: sysctl -n machdep.cpu.brand_string
```

Comment:
For future reference, this line prints the CPU information, which was the information motivating the switch to Appveyor.

Reply:
yes, this is a MacOS only command